shell/platform/tizen/flutter_tizen.cc

```diff
@@ -266,13 +269,30 @@ void FlutterDesktopViewOnPointerEvent(FlutterDesktopViewRef view,
 }
 
 void FlutterDesktopViewOnKeyEvent(FlutterDesktopViewRef view,
+                                  const char* device_name,
+                                  uint32_t device_class,
+                                  uint32_t device_subclass,
                                   const char* key,
                                   const char* string,
                                   uint32_t modifiers,
                                   uint32_t scan_code,
+                                  size_t timestamp,
                                   bool is_down) {
```

Comment:
Can't we provide API for NUI?
Added parameters are not required other than NUI. If we add a parameter to the common API, it would be uncomfortable when used in other view types.

Reply:
For reduce the uncomfortable, the added parameters have been moved and changed to the default parameters.
There are some APIs with UIFW dependencies, but no concrete plans yet.